components/EditSidebar/BaseCommit.js

```diff
@@ -36,6 +41,16 @@ class BaseCommit extends Component {
   render () {
     const { data: {loading, error, repo}, t } = this.props
     const commit = repo && repo.commit
+    let commitsBehind = null
+    if (repo) {
```

Comment:
mv logic into render function of loader – mv from `() => (<JSX />)` to `() => { doStuff(); return (<JSX />)}`